resolvers.ts

```diff
@@ -263,6 +264,19 @@ export const resolvers: Resolvers = {
       }
       return return_obj;
     },
+    ZipLink: async (root, args, context, info) => {
+      const res = await getFullResponse(`/workflow_runs/${args.workflowRunId}/zip_link.json`, args, context)
+      if (res.status !== 302) {
+        return {
+          url: null,
+          error: res.statusText,
+        };
+      }
+      const url = res.headers.get("location");
```

Comment:
Why is this not `res.url`?

Reply:
We don't need the url of the request in this case. We need the presigned s3 url that was returned in the header.

Reply:
Ah Interesting. As it stands I am getting this response from a valid workflowRunId: 
`{
  "data": {
    "ZipLink": {
      "url": null,
      "error": "OK"
    }
  }
}`

I see in the `res` object from `getFullResponse` the appropriate s3 url and can access it by console logging or outputting `res.url`. 

What would be the best way to adjust the code to make a successful call?